### DISCUSSION TITLE
Heroku- unable to connect [Jul 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2623] [Discord Threads]
> 🙋‍♂️ **npp**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@991355746979692544> could you please elaborate on the error observed? do double check your credentials by trying to directly connect to postgres database instead of trying the same from NocoDB  
#  
> 🙋‍♂️ **npp**  
#### tried to connect to postgres. so i chose new project from existing database  
#  
> 🙋‍♂️ **npp**  
#### any update  
#  
> 🙋‍♂️ **wkw**  
#### do u mean u create a project by connecting an external postgres database?  
#  
> 🙋‍♂️ **npp**  
#### correct  
#  
> 🙋‍♂️ **wkw**  
#### if it says invalid credentials, then it s not about connection issue  
#  
> 🙋‍♂️ **wkw**  
#### make sure your credential s correct  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/991356035099013232)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
